clients/attune/attune_log_parser.py

```diff
@@ -0,0 +1,80 @@
+import os
+import csv
+import re
+
+os.getcwd()
+os.chdir('test_logs/')
+
+log_file = 'System_2020-11-10_18-52-23.log'
+#Define regexes
+
+User_dictionary = {
+    'test' : 'test1'
+}
+
+event_done = re.compile('^Event:(?P<event>[^_]+)_Done')
+function_in_process = re.compile('^Instrument function (?P<state>.+) executing line (?P<line>\d+)$')
+event_start = re.compile('^Event:Starting_(?P<event>.+)$')
+user_login = re.compile('login')
+bubble_error = re.compile('^Data:New Bubble Detected')
+big_bubble = re.compile('BUBBLE SIZE GREATER THAN THRESHOLD!!!')
+aquisition_well = re.compile('^Acquisition initiated on well (?P<well>.+) Preload')
+aquisition_tube= re.compile('^Acquisition initiated$')
+
+
+with open(log_file) as log:
+    csv_log = csv.DictReader(log, delimiter=',')
+    csv_log.fieldnames = [
+        'TimeStamp', 
+        "LogType", 
+        "User", 
+        "Category", 
+        "Message", 
+        "unclearNumberString" ]
+
+    for line in csv_log:
+
+        
+        result=event_start.match(line["Message"])
+        if result != None:
+            function = result.group("event")
+            State = "Active"
```

Comment:
PEP8: lowercase S in state.

Also, this is a very Matlab-y way to initialize variables. It is fine that you are creating `State` inside the for loop and it does also work in Python, but it's not generally good practice. What happens if all of the if statements fail while parsing the entire file? Then the final print line will fail because `State` is undefined! It's good practice when doing something like this to explicitly declare `state = None` or some other sentinel, then check if it is not `None` at the end to handle this case.